Formula/j/jpeg-xl.rb

```diff
@@ -54,12 +55,17 @@ class JpegXl < Formula
   def install
     ENV.append_path "XML_CATALOG_FILES", HOMEBREW_PREFIX/"etc/xml/catalog"
     resources.each { |r| r.stage buildpath/"third_party"/r.name }
+    # jpegli.cmake looks for these `jpeg-turbo` headers at hardcoded path
```

Comment:
Can we fix this upstream instead?

Reply:
I made a mistake and assumed `jconfig.h.in` == `jconfig.h`, when it is not - the [.in file] has several constants awaiting to be filled out during configure stage: `#define JPEG_LIB_VERSION  @JPEG_LIB_VERSION@` while the delivered config header from `jpeg-turbo` reflects that particular instance: `#define JPEG_LIB_VERSION  80`

[.in file]:https://github.com/libjpeg-turbo/libjpeg-turbo/blob/2dfe6c0fe9e18671105e94f7cbf044d4a1d157e6/jconfig.h.in#L4

`jpegli` tries to provide an ABI/API compatible library to libjpeg, and seems to just reuse code from jpeg-turbo (who also provides an ABI/API compatible libjpeg), not 'including' that. These headers were already downloaded during a source build of libjxl, so I think it's totally reasonable for upstream to assume these exist?

I think the best way would be a resource block, downloading that exact `jconfig.h.in` to that exact place. The other two headers are untouched by building process and can copied from `jpeg-turbo` or as a resource too.